.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+.github/scripts               @notnmeyer
+.github/workflows             @notnmeyer
+projects/json-pointer-helpers @niclim
+projects/openapi-cli          @jaaprood
+projects/openapi-io           @niclim @jaaprood
+projects/openapi-utilities    @niclim @jaaprood
+projects/optic-ci             @niclim
```

Comment:
i forgot to add this and it's missing in monorail, but can we add `projects/optic @niclim` here as well